stylesheets/second.css

```
@@ -0,0 +1,102 @@
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+footer {
+    height: 7vh;
+    background-color: #dbd5d2;
+}
```

Adding some padding for the content to the left and right of your footer will make the content a little easier to read and see.

```suggestion
footer {
    height: 7vh;
    background-color: #dbd5d2;
    padding: 0 3% 0 3%;
}
```

You can actually add dimensions for padding on one line using the `padding: top left bottom right` syntax like I did in the suggestion or by specifically identifying a side like `padding-left: 3%`.  Both work just fine it's moreso the preference of the user.